pkgs/development/python-modules/pySMART_smartx/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pySMART.smartx";
+  version = "0.3.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16chrzqz3ykpkikfdi71z1g31hm8pij5gs9p6fsxjd6r3awxj1zr";
+  };
+
```

Comment:
checkPhase, pythonImportsCheck and maintainer missing.

Reply:
Is `pythonImportsCheck` not part of the normal `buildPythonPackage`?

```text
...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
pythonCatchConflictsPhase
pythonRemoveBinBytecodePhase
pythonImportsCheckPhase
Executing pythonImportsCheckPhase
/nix/store/7a1qrxxdqd4r2vw837fg9yps7vgpipmq-python3.8-pySMART-smartx-0.3.10
```

Reply:
I read the [documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) entry and I've added the required check.